code/modules/admin/admin.dm

```diff
@@ -805,3 +805,20 @@ var/global/floorIsLava = 0
 	qdel(frommob)
 
 	return 1
+
+/client/proc/adminGreet()
+	if(ticker && ticker.current_state == GAME_STATE_PLAYING)
+		var/string = pick(
+			"I welcome you to the server [key]!",
+			"Happy to see you [key]!",
+			"Ah, [key] logged, say hello!",
+			"The server welcomes you [key]!",
+			"Exceptional! [key] joined us, what an honour!",

```

Comment:
If there's one thing I can't stand more than someone being happy to see me, it's someone talking in an American accent.

:gb: God save the queen. :gb: